src/tzdb.rs

```diff
@@ -69,14 +70,14 @@ impl LocalTimeRecord {
     fn from_daylight_savings_time(info: &ZoneVariantInfo) -> Self {
         Self {
             is_dst: true,
-            offset: -info.offset.0,
+            offset: info.offset.0,
         }
     }
 
     fn from_standard_time(info: &ZoneVariantInfo) -> Self {
         Self {
             is_dst: false,
-            offset: -info.offset.0,
```

Comment:
The `ZoneVariantInfo` type was used primarily on the POSIX tz string. Inverting here is correct, because the time zones are inverted in the POSIX tz format. Australia is at -11 and EST is +5

Reply:
Got it.